### PR TITLE
LinkedIn OAuth Helper File

### PR DESCRIPTION
**Pull request by AbhishekShah212**
helper file to automate creation of LinkedIn OAuth Secret Credentials

Sorry I didn't submit the patch for the office365. I was out of country, but I did have time to make this :)

### CODE REVIEW

File: data/scripts/linkedinOAuthHelper.rb [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+serverConfig = {"name"=> "linkedin","type"=> "linkedin","options"=> {
+	  "api_access"=> api_access,
+	  "api_secret"=> api_secret,
+	  "oauth_token"=> oauth_token,
+	  "oauth_secret"=> oauth_secret   
+	} }  
+	
+puts serverConfig.to_json
+puts "\n\nwrote the above data to the linkedin JSON file\n\n"
+File.write('/root/.cartero/servers/linkedin.json', serverConfig.to_json)
```
Comment (FreedomCoder) [inline data/scripts/linkedinOAuthHelper.rb]: The script here is assuming you are running it under root user you should  pass ~/.cartero , use an env ENV["HOME"] and/or Dir.home.

Comment (FreedomCoder) [inline data/scripts/linkedinOAuthHelper.rb]: Using Dir.home will be best. 

```ruby
home = Dir.home
File.join(home, ".cartero") #=> /root/.cartero or C:/Users/MyUser/.cartero depending on the platform.
```

**Comment by AbhishekShah212** [comment]
Ahh nice catch! This should fix it.

**Comment by FreedomCoder** [comment]
**PR approved.** BTW, you might want to automate also your browser part. Using something like Mechanize so you can fully automate this process and no user interaction is needed. 👍

**Pull request merged**